projects/tinkerbell/tink/patches/0001-Remove-.ONESHELL-from-rules.mk.patch

```diff
@@ -0,0 +1,26 @@
+From 2972a803251a18e87ce92676cbac0aa079b32017 Mon Sep 17 00:00:00 2001
+From: Abhinav Pandey <abhinavmpandey08@gmail.com>
+Date: Tue, 3 May 2022 11:38:23 -0700
+Subject: [PATCH] Remove .ONESHELL from rules.mk
+
+Signed-off-by: Abhinav Pandey <abhinavmpandey08@gmail.com>
+---
+ rules.mk | 2 --
+ 1 file changed, 2 deletions(-)
+
+diff --git a/rules.mk b/rules.mk
+index a881c42..027f521 100644
+--- a/rules.mk
++++ b/rules.mk
+@@ -5,8 +5,6 @@ MAKEFLAGS += --no-builtin-rules
+ # This avoids a failure to then skip building on next run if the output is created by shell redirection for example
+ # Not really necessary for now, but just good to have already if it becomes necessary later.
+ .DELETE_ON_ERROR:
+-# Treat the whole recipe as a one shell script/invocation instead of one-per-line
+-.ONESHELL:
```

Comment:
What issue did this cause?

Reply:
Apparently the make version in builder-base has a bug where `.ONESHELL` and `. SHELLFLAGS` don't play well with each other. The error is something like `invalid option name`

Here's the stackoverflow question for reference https://stackoverflow.com/questions/68040084/oneshell-in-makefile-not-working-with-bash-on-amazon-linux-2-ami/68041602#68041602

Reply:
oh wow, interesting, kind of surprised we havent seen this before.  Well switch to an al22 base in the summer after they ga which has a newer make version.

Reply:
@pokearu mentioned he had seen this issue before when trying to build boots but yeah newer make version should fix this. Tried with `GNU Make 4.2.1` and no issues

Reply:
Tbh even an older make version would fix it I think. Make v3.82 is just terrible but AL2 wouldn't update